src/Cli/dotnet/commands/dotnet-workload/install/WorkloadManifestUpdater.cs

```diff
@@ -76,59 +78,59 @@ private IEnumerable<ManifestId> GetInstalledManifestIds()
             return manifests;
         }
 
-        private async Task UpdateAdvertisingManifestAsync(ManifestId manifestId, SdkFeatureBand featureBand, bool includePreviews)
+        private async Task UpdateAdvertisingManifestAsync(ManifestId manifestId, bool includePreviews)
         {
             string packagePath = null;
             string extractionPath = null;
+
+            var adManifestPath = GetAdvertisingManifestPath(_sdkFeatureBand, manifestId);
             try
             {
-                var adManifestPath = GetAdvertisingManifestPath(featureBand, manifestId);
-                packagePath = await _nugetPackageDownloader.DownloadPackageAsync(GetManifestPackageId(featureBand, manifestId), includePreview: includePreviews);
-                extractionPath = Path.Combine(_userHome, ".dotnet", "sdk-advertising-temp", $"{manifestId}-extracted");
-                Directory.CreateDirectory(extractionPath);
-                var resultingFiles = await _nugetPackageDownloader.ExtractPackageAsync(packagePath, extractionPath);
-
-                if (Directory.Exists(adManifestPath))
-                {
-                    Directory.Delete(adManifestPath, true);
-                }
-                Directory.CreateDirectory(Path.GetDirectoryName(adManifestPath));
-                FileAccessRetrier.RetryOnMoveAccessFailure(() => Directory.Move(Path.Combine(extractionPath, "data"), adManifestPath));
-
-                _reporter.WriteLine(string.Format(LocalizableStrings.AdManifestUpdated, manifestId));
+                packagePath = await _nugetPackageDownloader.DownloadPackageAsync(GetManifestPackageId(_sdkFeatureBand, manifestId), includePreview: includePreviews);
```

Comment:
Ideally this method would either throw a specific exception or have another way to indicate that no package was found, rather than having to catch any Exception that the method throws.